.travis.yml

```diff
@@ -17,12 +17,16 @@ jobs:
   include:
     - stage: Tests
       name: node 10
-      before_script: npm outdated || true
+      before_script:
+        - npm outdated || true
+        - node . driver install --source=npm appium-fake-driver@0.8.0
```

Comment:
do we need to pin the version here?

Reply:
i suppose it's safe to not pin it unless we run into a problem